operatorapi/operator_tenants.go

```diff
@@ -2917,3 +2925,37 @@ func getUpdateTenantYAML(session *models.Principal, params operator_api.PutTenan
 
 	return nil
 }
+
+func getTenantEventsResponse(session *models.Principal, params operator_api.GetTenantEventsParams) (models.EventListWrapper, *models.Error) {
+	ctx := context.Background()
+	client, err := cluster.OperatorClient(session.STSSessionToken)
+	if err != nil {
+		return nil, prepareError(err)
+	}
+	clientset, err := cluster.K8sClient(session.STSSessionToken)
+	if err != nil {
+		return nil, prepareError(err)
+	}
+	tenant, err := client.MinioV2().Tenants(params.Namespace).Get(ctx, params.Tenant, metav1.GetOptions{})
+	if err != nil {
+		return nil, prepareError(err)
+	}
+	events, err := clientset.CoreV1().Events(params.Namespace).List(ctx, metav1.ListOptions{FieldSelector: fmt.Sprintf("involvedObject.uid=%s", tenant.UID)})
+	if err != nil {
+		return nil, prepareError(err)
+	}
+	retval := models.EventListWrapper{}
+	for i := 0; i < len(events.Items); i++ {
+		retval = append(retval, &models.EventListElement{
+			Namespace: events.Items[i].Namespace,
+			LastSeen:  events.Items[i].LastTimestamp.Unix(),
+			Message:   events.Items[i].Message,
+			EventType: events.Items[i].Type,
+			Reason:    events.Items[i].Reason,
+		})
+	}
```

Comment:
```suggestion
	for _, event := range events {
		retval = append(retval, &models.EventListElement{
			Namespace: event.Namespace,
			LastSeen:  event.LastTimestamp.Unix(),
			Message:   event.Message,
			EventType: event.Type,
			Reason:    event.Reason,
		})
	}
```

would be more idiomatic